main.go

```diff
@@ -130,6 +130,7 @@ func fetchAndProcessTickets(
 		for _, listing := range filteredListings {
 			slog.Info(
 				"Found tickets for monitored event",
+				"configEvent", ticketConfig.Event,
 				"eventName", listing.Event.Name,
```

Comment:
Nice - maybe a slight tweak for easier identification what is what

Maybe something like (but will take better suggestions cos naming it hard!)
```suggestion
				"monitoredEvent", ticketConfig.Event,
				"matchedEvent", listing.Event.Name,
```